aws/components/network/setup.ftl

```diff
@@ -20,6 +20,19 @@
 
     [#local loggingProfile = getLoggingProfile(solution.Profiles.Logging)]
 
+    [#-- Flag that the flowlog configuration needs to be updated if enabled via flags --]
+    [#if
+        (
+            (environmentObject.Operations.FlowLogs.Enabled)!
+            (segmentObject.Operations.FlowLogs.Enabled)!
+            (solution.Logging.EnableFlowLogs)!false
+        ) &&
+        !(resources["flowLogs"]?has_content) ]
+        [@fatal
+            message="Flowlogs must now be explicitly configured on the network component"
+        /]
+    [/#if]
```

Comment:
Probably good to wrap this in a "template" subset to stop it from alerting in the resourcesets. Otherwise it will be hard to understand the context of this error

Reply:
done